README.md

```diff
@@ -1,8 +1,34 @@
 
 # penalties-file-notification-orchestrator
 
-This is a placeholder README.md for a new repository
+PENALTIES FILE NOTIFICATION ORCHESTRATOR has 2 endpoints
```

Comment:
Could you give a bit of explanation to what this does?
`This service accepts notifications to be sent to downstream services and processes batches of notifications at timed intervals, whilst handling the processing outcome of those notifications via callbacks.`
or something similar